control

```diff
@@ -691,6 +691,14 @@ class StepInit(Context, collections.Callable):
         return [subthing for subthing in subthings
                 if subthing in subtest_modules]
 
+# Grab a compressed copy of the audit log after every test
+# N/B: Autotest silently ignores any failures with this.
+audit_log = '/var/log/audit/audit.log'
+if os.path.isfile(audit_log):
+    command = 'gzip -9c %s' % audit_log
```

Comment:
Note: Compressing these goes from ~1MB per test down to ~60KB, so it's a significant reduction (times hundreds / thousands of tests).